tensorflow-framework/src/main/java/org/tensorflow/framework/initializers/Identity.java

```diff
@@ -33,12 +33,12 @@
  *     Identity&lt;TFloat32&gt; initializer =
  *             new org.tensorflow.framework.initializers.Identity&lt;&gt;(tf);
  *     Operand&lt;TFloat32&gt; values =
- *             initializer.call(tf.constant(Shape.of(2,2)), TFloat32.DTYPE);
+ *             initializer.call(tf.constant(Shape.of(2,2)), TFloat32.class);
  * </pre>
  *
  * @param <T> The TType for the call operation
  */
-public class Identity<T extends TType> extends BaseInitializer<T> {
+public class Identity<T extends TFloating> extends BaseInitializer<T> {
```

Comment:
This more restrictive, do we want that?

Reply:
Yes, because the `call` method has this code before:
```java
    if (!dtype.isFloating()) {
      throw new IllegalArgumentException("DataType must be a float type: " + dtype.name());
    }
```